content/kubeone/main/tutorials/creating-clusters/_index.en.md

```diff
@@ -885,6 +900,17 @@ cloudProvider:
 
     [Network]
     public-network = "NAT Network"
+  csiconfig: |
+    [Global]
```

Comment:
Don't we need credentials here as well? We have credentials here in E2E tests for example. Also, do we need parameters such as default-datastore and folder?

Reply:
Credentials are already there. Folder and datastore are something that we use but are not a requirement. 

The configuration in the documentation looks fine now. We may want to document other options in the CSI config but that's out of scope for this PR.